tools/virtualization/playing-around-with-various-vga-providers.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Playing around with various VGA providers  
+subsection: virtualization  
+order: 11  
+---
+
+# Playing around with various VGA providers
+
+> More information can be found here https://www.kraxel.org/blog/2019/09/display-devices-in-qemu/
+
+1. Download the Fedora Workstation ISO file from website and store it in a reference directory `cdromimg`.  
+   ```
+   https://getfedora.org
+   ```
+2. Execute the following command to know about the VGA providers available at disposal.  
+   ```
+   qemu-system-x86_64 -vga help
+   ```
+   ```
+   none                 no graphic card
+   std                  standard VGA (default)
+   cirrus               Cirrus VGA
+   vmware               VMWare SVGA
+   xenfb                Xen paravirtualized framebuffer
+   qxl                  QXL VGA
+   virtio               Virtio VGA
+   ```
+3. Start a virtual machine for playing around with various VGA providers for the live ISO and there is no need for a virtual disk image here.  
+   ```
+   qemu-system-x86_64 \
+     -boot menu=on \
+     -m 8192 \
+     -cpu max \
+     -smp 4 \
+     -cdrom cdromimg/<Fedora-Workstation-Live-x86_64-xx-y.z.iso> \
+     -accel kvm
+   ```
+4. Open up `Settings` inside the VM and head over to the `About` section. Information about the `Graphics` can be seen here.  
+5. Open up `Terminal` inside the VM and execute the `lspci` commands in regards with the current VGA controller to find more information on it.  
+
+We will test out each VGA provider one-by-one and list the information for each now.  
```

Comment:
Could you also describe the outcomes / implications for a regular user? IOW what's the aim / expected result of this testing / comparison?